helix-term/src/commands/lsp.rs

```diff
@@ -64,28 +64,78 @@ fn jump_to_location(
     align_view(doc, view, Align::Center);
 }
 
+/// Returns a string showing the symbol nested in its containing symbol if it exists.
+///
+/// # Arguments
+/// * `symbol` - the symbol that we're creating the string for
+///
+/// # Example
+/// If you have something like:
+/// ```
+/// mod foo {
+///     fn bar() {}
+/// }
+/// ```
+/// the result will be: `function  : foo → bar`
+fn get_symbol_string(symbol: &lsp::SymbolInformation) -> String {
+    let kind = match symbol.kind {
+        lsp::SymbolKind::FILE => "file      ",
```

Comment:
I have some idea but not sure if it's good or not. Why not do `[file] `?

Reply:
The downside of `[symbol kind]` is that it would take more horizontal space. It also feels a bit more cluttery (at least in my view). I think the ideal would be to eventually differentiate with color (which would also remove the need for the `→` separator altogether).